Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -132,6 +143,153 @@ TEST(BloomFilterTest,
   }
 }
 
+class BloomFilterGoldenTest : public ::testing::Test {
+ public:
+  void RunGoldenTest(const std::string& test_file) {
+    BloomFilter bloom_filter = GetBloomFilter(test_file);
+    std::string membership_result = GetMembershipResult(test_file);
+
+    for (size_t i = 0; i < membership_result.length(); i++) {
+      bool expectedResult = membership_result[i] == '1';
+      bool mightContainResult =
+          bloom_filter.MightContain(kGoldenDocumentPrefix + std::to_string(i));
+
+      EXPECT_EQ(mightContainResult, expectedResult);
+    }
+  }
+
+ private:
+  static const char* kGoldenDocumentPrefix;
+  JsonReader reader;
```

Comment:
Please make `JsonReader reader` a local variable in the method(s) that use it, rather than a member variable. We want to avoid any possible "bleeding" of information between method invocations and the performance gain of re-using an existing instance is negligible.